### PR TITLE
Added /processes params

### PR DESCRIPTION
**Pull request by doodomartins**

### CODE REVIEW

File: docs/satellite_endpoints.md [hunk 1]
```diff
@@ -240,7 +246,7 @@ const axios = require('axios')
 const TOKEN = 'YOUR_TOKEN'
 
 let endpoint = 'https://api.withleaf.io/services/satellite/api' +
-               '/fields/YOUR_ID/processes'
+               '/fields/YOUR_ID/processes?startDate=2020-06-03&endDate=2020-06-10'
 const headers = { 'Authorization': `Bearer ${TOKEN}` }
 
 axios.get(endpoint, { headers })
```
Comment (meyer1994) [inline docs/satellite_endpoints.md:252]: Better to use `axios` [params][1]:
```js
let endpoint = 'https://api.withleaf.io/services/satellite/api' + 
               '/fields/YOUR_ID/processes'

const params = { startDate: '2020-06-03', endDate: '2020-06-10' }
const headers = { 'Authorization': `Bearer ${TOKEN}` }

axios.get(endpoint, { headers, params })
```

[1]: https://github.com/axios/axios#example

File: docs/satellite_endpoints.md [hunk 2]
```diff
@@ -257,7 +263,7 @@ import requests
 TOKEN = 'YOUR_TOKEN'
 
 endpoint = ('https://api.withleaf.io/services/satellite/api'
-            '/fields/YOUR_ID/processes')
+            '/fields/YOUR_ID/processes?startDate=2020-06-03&endDate=2020-06-10')
 headers = {'Authorization': f'Bearer {TOKEN}'}
 
 response = requests.get(endpoint, headers=headers)
```
Comment (meyer1994) [inline docs/satellite_endpoints.md:269]: Better to use `requests` [params][1]:

```python
endpoint = ('https://api.withleaf.io/services/satellite/api'
            '/fields/YOUR_ID/processes')

params = {'startDate': '2020-06-03', 'endDate': '2020-06-10'}
headers = {'Authorization': f'Bearer {TOKEN}'}

response = requests.get(endpoint, headers=headers, params=params)
```

[1]: https://requests.readthedocs.io/en/master/user/quickstart/#passing-parameters-in-urls

**Review by meyer1994 — APPROVED** [review]

**Review by Dsbaule — APPROVED** [review]

File: docs/satellite_endpoints.md (hunk 1 above)
Comment (doodomartins) [reply docs/satellite_endpoints.md]: see 7b86c7e

File: docs/satellite_endpoints.md (hunk 2 above)
Comment (doodomartins) [reply docs/satellite_endpoints.md]: see 7b86c7e

**Review by GMBueno — APPROVED** [review]

**Pull request merged**